pkg/ioutils/writeflusher.go

```diff
@@ -59,34 +55,37 @@ func (wf *WriteFlusher) Flushed() bool {
 	// BUG(stevvooe): Remove this method. Its use is inherently racy. Seems to
```

Comment:
I'll address this separately as it requires a bit more work than I wanted to do in this PR.
See the changes to the `ContainerLogs` function to see what I plan to do to resolve this.

Currently this is being used in a few places by the image router and build.